exercises/queen-attack/queen-attack.test.ts

```diff
@@ -0,0 +1,75 @@
+import Queens from './queen-attack'
+
+describe('Queens', () => {
+    it('has the correct default positions', () => {
+        const queens = new Queens()
+        expect(queens.white).toEqual([0, 3])
```

Comment:
I don't think we should expose this. I'd rather see an error being thrown.

Reply:
as in, it white should probably be null

Reply:
I just enforced the necessity of using a parameter for queens initial position.